hearth/lib/hearth/retry.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require_relative 'retry/adaptive'
+require_relative 'retry/client_rate_limiter'
+require_relative 'retry/exponential_backoff'
+require_relative 'retry/retry_quota'
+require_relative 'retry/standard'
+
+module Hearth
+  module Retry
+    RetryToken = Struct.new(:retry_count, :retry_delay, keyword_init: true)
```

Comment:
What about shortening this name to `Token`?  (since its in the Retry namespace already)

Reply:
Yeah I can do that